Core/OxidEE_Events.php

```diff
@@ -153,9 +153,8 @@ private static function _createOrderTransactionTable()
             `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
             `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
             `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
-            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
-            `WDOXIDEE_STATE` enum('success','error','pending'),
-            `WDOXIDEE_PAYMENTMETHOD` varchar(32) NOT NULL,
```

Comment:
Why are you removing the payment method name?

Reply:
I serves no purpose in this table because we have to get the information from the payments table anyway in order to get a translated string of the name.